libft/ft_strsplit.c

```diff
@@ -41,7 +41,7 @@ static void	fill_array(char **array, char const *s, char c, int words)
 	}
 }
 
-static int	count_words(char const *s, char c)
+int	count_words(char const *s, char c)
```

Comment:
Same as above, if you want to create a new function in libft, better make it in a different file because afterwards it will be hard to find.